.github/workflows/main.yml

```diff
@@ -0,0 +1,149 @@
+name: CI
+on: push
+jobs:
+  lint-format-test:
```

Comment:
only one job ?
why not one for lint, one for format, etc ? It would be more practical to know the reason of a failure